docs/reference/objects/product/modifier.md

```diff
@@ -190,7 +190,7 @@ number
 {: .label .fs-1 }
 
 The remaining stock for the modifier. If the modifier has unlimited inventory this will return `nil`.
-- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), it will return the remaining stock for the modifier on the given date.
+exp- When accessed through an [experience slot]({% link docs/reference/objects/product/experience_slot.md %}), it will return the remaining stock for the modifier on the given date.
```

Comment:
👀 This `exp` seems like it doesn't belong here.

Reply:
Catalina eagle eyes at work, thank you so much 🙏

Reply:
<img width="50%" src="https://github.com/easolhq/easolhq.github.io/assets/10531930/20056122-f544-4b8c-a89d-e5ef5784d574" />